src/pages/ProfilePage/ProfilePage.js

```diff
@@ -62,7 +62,15 @@ const ProfilePage = ({
   inSelectMode,
 }) => {
   const scrollTo = useScroll();
+
   const [resumeLink, setResumeLink] = useState(null);
+  const [openForm, setOpenForm] = useState(false);
+
+  const handleUpdateContact = async values => {
+    await updateContact(values);
+    refreshContacts();
```

Comment:
You shouldn't have to call `refreshContacts` here as well, I don't think...`updateContact` should at the minimum handle the update

Reply:
I actually just realized why this call is probably here...I noticed you never updated the `reducer` to handle the `GET_CONTACT` or `UPDATE_CONTACT` result...that's actually a big problem. We should go over how to update that, but it should be a much better fix to this issue than adding `refreshContacts`

Reply:
added GET_CONTACT and UPDATE_CONTACT to reducer. Removed refreshContacts() in handleUpdateContact().